src/_pytask/live.py

```diff
@@ -185,9 +185,7 @@ def pytask_execute_task_log_end(self, report: ExecutionReport) -> bool:
         self.update_reports(report)
         return True
 
-    def _generate_table(
-        self, reduce_table: bool, sort_table: bool, add_caption: bool
-    ) -> Table | None:
+    def _generate_table(self, reduce_table: bool, add_caption: bool) -> Table | None:
```

Comment:
I think this has to stay. The goal of this PR is to change the sorting of the final table, right?

Because, the current behavior is that the table is unsorted and the last 15 executed tasks are shown. But, when the execution finishes, the whole table is shown and it is sorted.

I thought you only wanted to change whether the table is sorted in the end. Because during the execution a sorted table does not make sense.

Reply:
Yes, you're completely right.